Controller/CRUDController.php

```diff
@@ -465,7 +465,17 @@ public function batchAction()
         if (count($idx) > 0) {
             $this->admin->getModelManager()->addIdentifiersToQuery($this->admin->getClass(), $query, $idx);
         } elseif (!$allElements) {
-            $query = null;
+            $this->addFlash(
+                'sonata_flash_info',
+                $this->trans('flash_batch_no_elements_processed', [], 'SonataAdminBundle')
```

Comment:
Why not using the existing key `flash_batch_empty` for this?

Reply:
The output message of `flash_batch_empty` is telling to user that there is no selection. But, in this case, the user has selected items. So the displayed message is not the right one.